cpp/csp/cppnodes/statsimpl.h

```diff
@@ -1357,7 +1357,7 @@ class Rank
     private:
         ost<std::less<double>> m_mintree;
         ost<std::greater<double>> m_maxtree;
-        double m_lastval;
+        double m_lastval = std::numeric_limits<double>::quiet_NaN();
```

Comment:
@AdamGlustein

Reply:
Yup this is fine (as you mention it won't be used, because if there is no data when the calc is triggered then NaN will be returned, not m_lastVal).